cleanlab/filter.py

```diff
@@ -290,6 +294,28 @@ class 0, 1, ..., K-1.
             multi_label=multi_label,
             return_indices_of_off_diagonals=True,
         )
+    # TODO: (delete once read and/or make a comment) -- We don't re-use confident_joint inside this method in num_label_issues(confident_joint=confident_joint), bcconfident_joint is calibrated and by default the method uses an uncalibrated confident_joint. Delete this todo once confirmed this is correct.
+    if filter_by in ["low_normalized_margin", "low_self_confidence"]:
+        from cleanlab.rank import get_label_quality_scores
+        from cleanlab.count import num_label_issues
+
+        # TODO: consider setting adjust_pred_probs to true based on benchmarks (or adding it kwargs, or ignoring and leaving as false by default)
+        scores = get_label_quality_scores(
+            labels,
+            pred_probs,
+            method=filter_by[7:],
+            adjust_pred_probs=False,
```

Comment:
should `adjust_pred_probs` value come from `rank_by_kwargs["adjust_pred_probs"]` if that was specified by user? Or always restrict to False?

Reply:
I'm not sure what the benchmarks suggested in terms of it being on by default for these methods. If no information, would be nice for it to be added to rank_by_kwargs["adjust_pred_probs"] with a default value of `False`

Reply:
I think having the default adjust_pred_probs be False is the way to go. I am more asking whether we want to support allow the user to filter_by low_self_confidence but with self_confidence being computed with adjusted pred_probs. There is no way to do this in your current PR AFAICT, one can only find the label issue mask with unadjusted self_confidence/normalized_margin and then choose to later use adjusted pred_probs when ranking the datapoints with label issues

Reply:
passing this off to you but let me know if you still want me inolved @jwmueller